src/main/kotlin/com/bardsoftware/papeeria/backend/BaseServer.kt

```diff
@@ -175,6 +176,10 @@ fun start(arg: BaseServerArgs, service: BackendService, serverName: String) = ma
     server
   }
   service.pubsub?.let {
+    if (arg.sub == null) {
+      System.err.println("Missing subscription name.")
+      exitProcess(1)
+    }
     LOG.info("Listening to PubSub subscription ${arg.sub!!}")
```

Comment:
Looks like checking for nullity here is redundant, if you are doing it above

Reply:
refactored a little bit